FastRGF/README.md

```diff
@@ -120,15 +120,20 @@ make install
 -  **discretize.sparse.max_features**: Controls the maximum number of selected features. Typical range is `[1000, 10000000]`. Default value is `80000`. Meant for being used with sparse data.
 -  **discretize.sparse.min_occurrences**: Controls the minimum number of occurrences for a feature to be selected. Default value is `5`. Meant for being used with sparse data.
 
-# 5. Contact
+# 5. Known Issues
+
+- FastRGF crashes if training dataset is too small (#data < 28). (`rgf#92 <https://github.com/RGF-team/rgf/issues/92>`__)
+- FastRGF does not provide any built-in method to calculate feature importances. (`rgf#109 <https://github.com/RGF-team/rgf/issues/109>`__)
+
```

Comment:
This is `MD`-file, not `RST` one, so the link should be `[title](href)`.

![image](https://user-images.githubusercontent.com/25141164/52915674-050c3b80-32e8-11e9-94c9-d05f0507cc41.png)